scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -279,14 +282,14 @@ def export_standalone_parse_args(o3de_context: exp.O3DEScriptExportContext, expo
                                               parser=parser,
                                               key=exp.SETTINGS_SEED_LIST_PATHS.key,
                                               dest='seedlist_paths',
-                                              description='Path to a seed list file for asset bundling. Specify multiple times for each seed list.',
+                                              description='Path to a seed list file for asset bundling. Specify multiple times for each seed list. You can also specify wildcard patterns.',
                                               is_path_type=True)
 
         export_config.add_multi_part_argument(argument=['-sf', '--seedfile'],
                                               parser=parser,
                                               key=exp.SETTINGS_SEED_FILE_PATHS.key,
                                               dest='seedfile_paths',
-                                              description='Path to a seed file for asset bundling. Example seed files are levels or prefabs.',
+                                              description='Path to a seed file for asset bundling. Example seed files are levels or prefabs. You can also specify wildcard patterns.',
```

Comment:
Wait is a seed file a .spawnable or a .seed? This makes it sounds like it's a .spawnable.